src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -163,7 +163,7 @@
     'SCCF_chan_wavelen': ('sensorCentralFrequency', 'double'),
     'QI_with_FC': ('percentConfidenceWithForecast', 'float'),
     'QI_without_FC': ('percentConfidenceWithoutForecast', 'float'),
-    'Data_Vertical_Velocity': ('instantaneousAltitudeRate', 'float'),
```

Comment:
I don't know if this change is correct, since it is needed for aircraft @nicholasesposito any comments?